README.md

```diff
@@ -30,25 +30,60 @@ Daily, a zip file is created, containing a CSV file with all events using the fo
 
 The format is described here: http://data.gdeltproject.org/documentation/GDELT-Data_Format_Codebook.pdf
 
-### TPCDS
+The `iceberg/datasets/gdelt/fetch.sh` script download the GDELT file (eventually using a given date):
+
+```
+./iceberg/datasets/gdelt/fetch.sh 20240114
+```
+
+The script downloads and extracts the GDELT ready to be used.
 
 ## Use cases
 
 ### Data Ingestion to Iceberg using Spark
 
-* `gdelt/spark/di`
+`iceberg/usecases/injection` contains `CreateTable` class. This class creates an Iceberg table to store the GDELT events.
 
-### Q1: Events By Location
+You first have to execute `CreateTable` to create the table used by other use cases.
 
-* `gdelt/spark/q1`
+You can build a uber jar using:
 
-This query extracts all events for a specific location, using Spark engine.
+```
+mvn clean install
+```
+
+You now have the `iceberg/usecases/injection/target/injection-1.0-SNAPSHOT.jar` uber jar.
+
```

Comment:
Should we add a driver with all the available commands? Either we can prompt all the available commands or we can define Command Options to be selected ?  thoughts ?